resources/js/views/settings/users/View.vue

```diff
@@ -0,0 +1,510 @@
+<template>
+  <component
+    v-bind:is="config.type === 'profile' ? 'b-container' : 'div'"
+    v-bind:class="{ 'mt-3': config.type === 'profile', 'mb-5': config.type === 'profile' }"
+  >
+    <component
+      v-bind:is="config.type === 'profile' ? 'b-card' : 'div'"
+      v-bind:class="{ 'p-3': config.type === 'profile', border: config.type === 'profile', 'bg-white': config.type === 'profile' }">
+      <h3>
+        {{ config.id === 'new' ? $t('settings.users.new') : (
+          config.type === 'profile' ? $t('app.profile') :
+            $t(`settings.users.${config.type}`, { firstname: model.firstname, lastname: model.lastname })
+        ) }}
+      </h3>
+      <hr>
+
+      <b-form @submit='saveUser'>
+        <b-container :fluid='true'>
+          <b-form-group
+            label-cols-sm='3'
+            :label="$t('settings.users.firstname')"
+            label-for='firstname'
+            :state='fieldState("firstname")'
+          >
+            <b-form-input
+              id='firstname'
+              type='text'
+              v-model='model.firstname'
+              :state='fieldState("firstname")'
+              :disabled="isBusy || config.type === 'view' || model.authenticator !== 'users'"
```

Comment:
I would like to disable the editing of firstname, lastname and email of some users. Could we add this as a group permission e.g.: users.update_own_attributes

Reply:
We could do that. Should this be additionally if the user are database users? In other cases this don't make sense if we can't write to the service back, like it is the case for ldap.

Reply:
Yes, only for db users. LDAP user attributes should be updated on each login.

Reply:
This should only be used for the profile page correct?

Reply:
Yes, the other user editing pages should only be accessible for users with permission to always edit these attributes. The idea behind my comment was, so no student can change their name to something that can't be used to identify them. The information should always come from a trustful source (admin users or ldap). For some users we could grant this permission based on trusting them.

Reply:
see 3a4780d